src/infra/core/src/repos/dataset_repository_local_fs.rs

```diff
@@ -484,7 +483,7 @@ impl DatasetStorageStrategy for DatasetSingleTenantStorageStrategy {
     }
 
     fn get_datasets_by_owner<'s>(&'s self, account_name: AccountName) -> DatasetHandleStream<'s> {
-        if account_name == self.current_account_subject.account_name {
+        if account_name.eq(&self.current_account_subject.account.login) {
```

Comment:
why this change?

Reply:
Wasn't sure on the idiomatic way. Corrected here and in one more place